05-UI-ReactJS/source/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import {handleMovies} from "./redux/reducer";
+import {initializeState} from "./redux/action";
+import store from "./store";
+import Application from "./movieApp"
+
+store.dispatch(initializeState());
```

Comment:
Don't seems this is the best way of initialize the store... You should do this inside the store